### DISCUSSION TITLE
Support for the Oracle Database?

### DISCUSSION THREAD

**Discussion by JonOfUs** [thrillfall/nextcloud-gpodder#77] [Ideas]
Hello,

today I learned that the Oracle Database is supported by Nextcloud after all. There are only a few mentions of this in the documentation, especially nothing in the installation instructions. It seems like this database type is only available in [enterprise subscriptions of Nextcloud](https://docs.nextcloud.com/server/24/admin_manual/installation/system_requirements.html?highlight=oracle#server). So OCI being in the CI matrix may have been intended and useful.

Now to the question of this discussion: **should we add support for the _Oracle Database 11g_**?

I removed the OCI tests because subscription integration tests always failed in this DB (probably) due to DateTime formats (https://github.com/thrillfall/nextcloud-gpodder/pull/73#issuecomment-1137563728). So we would need to change this format or add a special case only for OCI databases in order for gpoddersync to work with this database type.

Is this adjustment worth the trouble, or is it even necessary? I don't know anything about enterprise subscriptions, so I don't know how relevant this is.

A possible workaround could be to put into the [app metadata](https://docs.nextcloud.com/server/stable/developer_manual/app_development/info.html) that only MySQL, SQLite and PostgreSQL are supported, there is an option for database dependencies under _dependencies/database_.

**Comment by thrillfall** [comment]
I say dump it. If there are Oracle users they can provide a fix having enterprise resources :-)
